logs/README.md

```diff
@@ -68,3 +76,12 @@ Returns all logs according to the provided queries and path/named-route param.
       "message": "Something went wrong. Please contact admin"
     }
     ```
+  - **Code:** 503
+    - **Content:**
+    ```json
+    {
+      "statusCode": 503,
+      "error": "Internal Server Error",
+      "message": "Something went wrong. Please contact admin"
+    }
+    ```

```

Comment:
why are handling internal server error here if it is defined above as 500?, any specific reason??

Reply:
no, error 5xx is same so just keeping it